src/editor/codemirror/language-server/view.ts

```diff
@@ -38,13 +40,11 @@ class LanguageServerView extends BaseLanguageServerView implements PluginValue {
     // Is there a better way to do this? We can 't dispatch at this point.
     // It would be best to do this with initial state and avoid the dispatch.
     setTimeout(() => {
-      const initialDiagnostics = this.client.currentDiagnostics(this.uri);
-      view.dispatch(
-        setDiagnostics(
-          view.state,
-          diagnosticsMapping(this.view.state.doc, initialDiagnostics)
-        )
+      const diagnostics = diagnosticsMapping(
+        view.state.doc,
```

Comment:
Should this also be `this.view.state.doc`

Reply:
It doesn't really matter, view has a stable identity. For consistency I've made all access in the constructor use the parameter rather than the field. Its correctness is then more obvious as you don't have to consider when the field assignment happens (though it's pretty clearly in super).

Reply:
3688a534f641f2732e0671d4d813ccdfc8681eec